README.md

```diff
@@ -1,6 +1,20 @@
 Absolute
-=======
+=============
+
+### A web-based order platform like siren-order for Starbucks.
```

Comment:
IMHO, this might be unnecessary because this sentence is duplicated with follow-up section.
WDYT?